src/GoodsList.tsx

```diff
@@ -8,7 +8,11 @@ type Props = {
 export const GoodsList: React.FC<Props> = ({ goods }) => (
   <ul>
     {goods.map(good => (
-      <li key={good.id} data-cy="good">
+      <li
+        key={good.id}
+        data-cy="good"
+        style={{ color: `${good.color}` }}
```

Comment:
```suggestion
        style={{ color: good.color }}
```
You don't need template string here